tests/testthat/test-attach_messaging.R

```diff
@@ -14,9 +14,3 @@ test_that("highlights dev versions in red", {
     highlight_version(c("1.0.0", "1.0.0.9000", "0.9000.0.9000", "1.0.0-rc"))
   })
 })
-
-test_that("useful conflicts message", {
-  expect_snapshot({
-    crane_conflicts(c("testthat", "gtsummary"))
```

Comment:
I had to remove this because it is not showing the same conflicts on integration tests, I do not know exactly why. coverage is good also without this (it would be the `only=` parameter). Should I try to make it work anyway @ddsjoberg?

Reply:
in other words the issue is that there are no conflicts on integration tests for some reason, while there are some in local (tried with different packages too)

Reply:
i am not sure either 🤷🏼